src/components/SearchInputHeader.js

```diff
@@ -0,0 +1,11 @@
+ import React from 'react';
+
+class SearchInputHeader extends React.Component {
+	render () {
+		return (
+			<th>{this.props.header}</th>
```

Comment:
In all honesty, I wouldn't make a component if it's this small. But it's not necessarily a problem.

I think it would benefit from being made into a functional component

Reply:
i think thats a problem of table layout. if I did something similar from scratch, i wouldn't make it a separate component, just in this case I somehow decided it will look nicer

Reply:
😆 yep, one of the things on my todo list is to change the example so that it doesn't use `<table>` for layout